### ISSUE TITLE
[HL] Music gets abruptly cut during level transisions

### ISSUE THREAD

**Issue by HalfWayLambda** [ValveSoftware/halflife#2890]
This is something that has been already fixed, sort of. Basically, every time you go through a level transision the music gets abruptly cut and doesn't play again. But here's the thing: this is fixed in the default game, but not for everything else. That means that the expansion packs of Opposing Force and Blue Shift also have this problem (but it might be that using the beta branch for those might fix this), but then again this problem persists with any other mod. Even if a mod uses the default client and server dlls found inside the valve folder, it still gets abruptly cut. So those dlls most likely don't have to do with this problem. It's most likely related to the engine in some way.
A similar issue exists with the raw mouse input option, it works for the default game but not for everything else where your view gets limited to ~30 degrees of rotation and you can't play with it on, but i'm getting off-topic.
Also, something else that i've noticed is quite the opposite problem: the music gets abruptly cut _after_ it begins after a level transision, or when launching a new map in general. This is most noticable at the start of the game in the tram ride, where the game's supposed to play the vague voices song, but it plays for only 0.5 seconds and then just stops. This also happens just after the Unforeseen Consequences chapter begins, where it plays the space ocean song for 0.5 seconds again and then it also cuts. But then again, this issue also happens in every mod except for the default game (and Blue Shift and Opposing Force are probably also immune to this in the beta branch, but i'm not 100% sure about that). From what I can remember, this specific bug is caused because the game starts playing the music while the map is loading, and when it's fully loaded, it stops.
Sorry if I was confusing, i'll explain it more simply if I wasn't clear enough.

I don't think this is related to the problem, but i'm running on Windows 10 version 1909, and here's the debug info from the `version` console command (running on the beta branch):
```
Protocol version 48
Exe version 1.1.2.2/Stdio (valve)
Exe build: 15:17:55 Jul 24 2019 (8308)
```

**Comment by kisak-valve** [comment]
Hello @HalfWayLambda, this issue is already being tracked at #570. Feel free to add your experience to the existing issue report.

Closing as a duplicate.
